### PR TITLE
feature/site-redesign-1

### PR DESCRIPTION
**Pull request by 3bobchen**

### CODE REVIEW

**Comment by francojreyes** [comment]
The background just cuts off - maybe only use it on the home screen?
image

on mobile too:
image

File: app/layout.tsx [hunk 1]
```diff
@@ -23,14 +25,16 @@ export default async function RootLayout({
   return (
     <html lang="en">
       <body>
+        <NavBar/>
```
Comment (francojreyes) [inline app/layout.tsx:28]: Feels like these should be under the `ThemeRegistry` so it can use theme stuff (if it needs to)

File: app/page.tsx [hunk 2]
```diff
@@ -1,170 +1,260 @@
...
-  }, [])
-  const texts = ['Developers', 'Enthusiasts', 'Change makers', 'Engineers', 'Creators', 'Mathematicians']
   return (
-    <Stack
-      pt={10}
-      spacing={{ xs: 4, sm: 10 }}
-      alignItems="center"
-      width="100%"
-      height="100%"
+    <Box
```
Comment (francojreyes) [inline app/page.tsx:79]: ~~Box display: 'flex'~~ Stack !!! Also, system props over `sx`
```tsx
<Stack direction="column" pt={2} px={2} width="100%" maxWidth={1300} m="auto" overflow="hidden">
```

File: data.ts [hunk 3]
```diff
@@ -38,11 +38,11 @@ import { StaticImport } from "next/dist/shared/lib/get-img-props";
...
 // Probably shouldn't be touched
 export const navData = [
-  { text: "About Us", href: "/about-us" },
-  { text: "Our Projects", href: "/our-projects" },
+  { text: "About", href: "/about-us" },
+  { text: "Projects", href: "/our-projects" },
   { text: "Starlight", href: "/starlight" },
-  { text: "Get Involved", href: "/get-involved" },
-  { text: "Supporters", href: "/supporters" },
+  { text: "Get involved", href: "/get-involved" },
```
Comment (francojreyes) [inline data.ts:44]: Why did you uncapitalise this lol

File: app/page.tsx [hunk 4]
```diff
@@ -1,170 +1,260 @@
...
+          scrollbarWidth: 'none',
+        }}
+      >
+        <Stack
+          spacing={{xs: 3, md: 5}}
+          direction={{xs: "column", md: "row"}}
+          sx={{display: "flex", width: '100%'}}
+          flexDirection={{xs: "column", md: "row"}}
+        >
+          {projectData.map((props, idx) => <LinkCard key={props.name} {...props} order={idx}/>)}
```
Comment (francojreyes) [inline app/page.tsx:144]: Lowkey I think Chaos should not be on the homepage (not functional), and possibly not Jobsboard either (practically not functional)

File: data.ts (hunk 3 above)
Comment (3bobchen) [reply data.ts:44]: gramatical correctness - but do u prefer the capitalised version?

File: app/page.tsx (hunk 4 above)
Comment (3bobchen) [reply app/page.tsx:144]: ok

File: data.ts (hunk 3 above)
Comment (francojreyes) [reply data.ts:44]: Yeah I mean headings are generally capitalised right?

**Comment by 3bobchen** [comment]
> The background just cuts off - maybe only use it on the home screen? image
> 
> on mobile too: image

Agree - done

File: app/layout.tsx (hunk 1 above)
Comment (3bobchen) [reply app/layout.tsx]: done

File: app/page.tsx (hunk 2 above)
Comment (3bobchen) [reply app/page.tsx]: done. - but wdum, can box not have dispay flex or is a box display flex === stack?

**Review by francojreyes — APPROVED** [review]

File: app/page.tsx (hunk 2 above)
Comment (francojreyes) [reply app/page.tsx]: box display flex === stack

**Pull request merged**